APPSEC-60862 - General update on ai guard public docs (#34867)
DataDog/documentation: content/en/security/ai_guard/onboarding.md

To restrict access to AI Guard spans for specific users, you can use [Data Access Control][7]. Follow the linked instructions to create a restricted dataset, scoped to **APM data**, with the `resource_name:ai_guard` filter applied. Then, you can grant access to the dataset to specific roles or teams.

> ## Try the AI Guard Playground {#playground}

Reviewer: ```suggestion
## Evaluate conversations in AI Guard Playground {#playground}
```
Thought we could phrase this to be a little more purpose-based?

<div class="alert alert-info">You can't see data in Datadog for evaluations performed directly using the REST API.</div>

> ## Configure AI Guard policies {#configure-policies}

Reviewer: I would consider moving this section higher on the page. Users might assume that `DENY` or `ABORT` actions automatically get blocked, so they might want to change this configuration earlier in the process.

Author: Where do you think it's the best? Because it doesn't make sense to me to put it before the SDKs integration as people need to use the SDK to really block

Reviewer: How would you feel about putting it as a subsection under Setup?

AI Guard provides two settings to control how evaluations are enforced and how sensitive threat detection is.

### Blocking policy {#blocking-policy}

> By default, AI Guard evaluates conversations and returns an action (`ALLOW`, `DENY`, or `ABORT`) but does not block requests. To enable blocking so that `DENY` and `ABORT` actions actively prevent unsafe interactions from proceeding, configure the [blocking policy][20] for your services.

Reviewer: ```suggestion
By default, AI Guard evaluates conversations and returns an action (`ALLOW`, `DENY`, or `ABORT`) but does not block requests. To enable blocking so that `DENY` and `ABORT` actions actively prevent unsafe interactions from proceeding for a service, go to the **AI Guard** > **Settings** > [**Services**][20] and turn on the **Blocking enabled** toggle beside its name.
```

Reviewer: Not sure about the wording here? The page and table are both called Services, so I thought this was just for services. Unless this is coming in a future release, I don't see an option to, say, turn on blocking for everything in the prod environment.

You can configure blocking at different levels of granularity, with more specific settings taking priority:
1. **Organization-wide**: Apply a default blocking policy to all services and environments.
2. **Per environment**: Override the organization default for a specific environment.
3. **Per service**: Override the organization default for a specific service.
4. **Per service and environment**: Override all of the above for a specific service in a specific environment (for example, enable blocking in production but not in staging).

### Evaluation sensitivity {#evaluation-sensitivity}

AI Guard assigns a confidence score to each threat category it detects (for example, prompt injection or jailbreaking). The [evaluation sensitivity][21] setting controls the minimum confidence score required for a threat to be flagged. It is a value between 0.0 and 1.0, with a default of 0.5.

> A lower value increases sensitivity: AI Guard flags threats even when the confidence is low, surfacing more potential attacks but also more false positives. A higher value decreases sensitivity: AI Guard only flags threats when the confidence is high, reducing noise but potentially missing some attacks.

Reviewer: ```suggestion
AI Guard assigns a confidence score to each threat category it detects (for example, prompt injection or jailbreaking). You can control the minimum confidence score required for AI Guard to flag a threat by going to **AI Guard** > **Settings** [**Evaluation Sensitivity**][21].

Evaluation sensitivity is a value between 0.0 and 1.0, with a default of 0.5.
- A **lower** value **increases** sensitivity: AI Guard flags threats even when the confidence is low, surfacing more potential attacks but also more false positives.
- A **higher** value **decreases** sensitivity: AI Guard only flags threats when the confidence is high, reducing noise but potentially missing some attacks.
```
